libs/domains/clusters/feature/src/lib/cluster-installation-guide-modal/cluster-installation-guide-modal.tsx

```diff
@@ -0,0 +1,68 @@
+import download from 'downloadjs'
+import { Button, Callout, ExternalLink, Icon, IconAwesomeEnum, Skeleton } from '@qovery/shared/ui'
+import { useInstallationHelmValues } from '../hooks/use-installation-helm-values/use-installation-helm-values'
+
+export interface ClusterInstallationGuideModalProps {
+  clusterId: string
+  organizationId: string
+  onClose: () => void
+}
+
+export function ClusterInstallationGuideModal({
+  clusterId,
+  organizationId,
+  onClose,
+}: ClusterInstallationGuideModalProps) {
+  const { data: installationHelmValues, isLoading } = useInstallationHelmValues({ organizationId, clusterId })
+  const downloadInstallationValues = () => {
+    download(installationHelmValues ?? '', `cluster-installation-guide-${clusterId}.yaml`, 'text/plain')
+  }
+
+  return (
+    <div className="p-6 flex flex-col gap-6">
+      <h2 className="h4 text-neutral-400 max-w-sm truncate">Installation guide</h2>
+
+      <ol className="ml-4 list-outside list-decimal" type="1">
+        <li className="mb-6">
+          <span>Save the following yaml, it contains the Qovery configuration assigned to your cluster.</span>
+          <br />
+          <span className="inline-block mt-2">
+            <Skeleton height={40} width={110} show={isLoading}>
```

Comment:
I think the skeleton don't have a good usage here, the fetch to get the file should be only on the click and you can use the `loading` props on the `Button` component. It will be visually more pleasant

Reply:
https://github.com/Qovery/console/pull/1117/commits/d5cf6c5f3f2e35bec33715129a7584c26ea35b53

Reply:
it missing the `loading` props on the button?